cspell.config.yaml

```diff
@@ -21,10 +21,16 @@ words:
   - mcap
   - nsec
   - nsecs
+  - pipenv
   - proto
   - protobuf
+  - pypi
+  - pyright
+  - pytest
+  - rglob
```

Comment:
maybe put these in the "overrides" section below under a python specific set of rules